.travis.yml

```diff
@@ -1,6 +1,9 @@
 ---
 language: python
 dist: xenial
+branches:
```

Comment:
ooooh. super nice. didnt know about this config. had double builds on all pushes